apps/web/pages/team/[slug].tsx

```diff
@@ -252,20 +257,22 @@ function TeamPage({ team, isUnpublished, markdownStrippedBio, isValidOrgDomain }
 }
 
 export const getServerSideProps = async (context: GetServerSidePropsContext) => {
-  const ssr = await ssrInit(context);
   const slug = Array.isArray(context.query?.slug) ? context.query.slug.pop() : context.query.slug;
   const { isValidOrgDomain, currentOrgDomain } = orgDomainConfig(
     context.req.headers.host ?? "",
     context.params?.orgSlug
   );
   const flags = await getFeatureFlagMap(prisma);
   const team = await getTeamWithMembers({ slug, orgSlug: currentOrgDomain });
+  const ssr = await ssrInit(context, {
+    noI18nPreload: isValidOrgDomain && !team?.parent,
```

Comment:
no i18n string preloading when we are in a valid organization and it is the org profile, not a subteam